tests/functional/firefox/whatsnew/test_whatsnew.py

```diff
@@ -29,10 +29,7 @@ def test_send_to_device_fails_when_missing_required_fields(base_url, selenium):
 
 @pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
 @pytest.mark.nondestructive
-def test_firefox_rocket_send_yourself(base_url, selenium):
+def test_firefox_lite_qr_code(base_url, selenium):
     page = FirefoxWhatsNewPage(selenium, base_url, locale='id').open()
-    assert page.send_to_device.is_displayed
-    send_to_device = page.send_to_device
-    send_to_device.type_email('success@example.com')
-    send_to_device.click_send()
-    assert send_to_device.send_successful
+    assert not page.send_to_device.is_displayed
+    assert page.is_qr_code_displayed
```

Comment:
This test is failing because it's for a QR code defined on a different page to the one being tested here.

I'd suggest using a new property here, like `assert page.is_firefox_lite_qr_code_displayed`.

You can then define that property in [whatsnew.py](https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/whatsnew/whatsnew.py) using:

```python
_firefox_lite_qr_code_locator = (By.CSS_SELECTOR, '.lite-qrcode-container img')

@property
def is_firefox_lite_qr_code_displayed(self):
    return self.is_element_displayed(*self._firefox_lite_qr_code_locator)